docs/Guides/IAP-Integration.md

```diff
@@ -31,142 +31,128 @@ Each provider has their own specific setup and Integration.
-<br />
+CloudPay will verify the receipts sent from the App with the Roku Validate-transaction API. The Partner API key is required as part of the request.
+
 ##### Notification service
-Publishers can subscribe to Roku Pay push notifications by providing the Cloud Pay endpoint on the [Roku Pay Web Services](https://developer.roku.com/docs/developer-program/roku-pay/quickstart/setting-up-web-services.md#push-notification-url) page in the Developer Dashboard.  However, this is not guaranteed and should not be relied on for all subscription updates.
-<br />
+Publishers can subscribe to Roku Pay push notifications by providing the CloudPay endpoint on the [Roku Pay Web Services](https://developer.roku.com/docs/developer-program/roku-pay/quickstart/setting-up-web-services.md#push-notification-url) page in the Developer Dashboard.  However, this is not guaranteed and should not be relied on for all subscription updates.
+
+```https://{client-cloudpay-instance}.streamamg.com/notification/v1/roku```
+
 ##### StreamAMG Requirements:
-* **Partner API Key**: Private Key is used to identify the developer issuing the request.  This is added via the Cloud Pay Admin console and can be accessed from the [Roku Pay Web Services](https://developer.roku.com/docs/developer-program/roku-pay/implementation/roku-web-service.md#roku-pay-api-key) page in the Developer Dashboard.
+- **Partner API Key**: Private Key is used to identify the developer issuing the request.  This is added via the CloudPay Admin console and can be accessed from the [Roku Pay Web Services](https://developer.roku.com/docs/developer-program/roku-pay/implementation/roku-web-service.md#roku-pay-api-key) page in the Developer Dashboard.
 
-* **Cloud Pay endpoint**: Endpoint details will be provided as part of the setup.
+- **CloudPay endpoint**: CloudPay endpoint will be provided as part of the setup.
 
 
 ## Guide
 
 ### Amazon
 
-1. Create a ***Subscriptions*** based IAP via the ***Amazon App Store***. Developer details can be found [here.](https://developer.amazon.com/docs/in-app-purchasing/iap-overview.html)
-
-2. Add Cloud Pay notification end point (supplied by StreamAMG) to the ***Real-Time Notifications*** section in the App Store.
-
-3. Create Cloud Pay packages via the Admin console that match the IAP packages in terms of cost, type (renewable or non-renewable) and subscription length.
+1. Create a ***Subscriptions*** based IAP via the ***Amazon App Store***. Developer details can be found [here.](https://developer.amazon.com/docs/in-app-purchasing/iap-overview.html).
+2. Add CloudPay notification end point (supplied by StreamAMG) to the ***Real-Time Notifications*** section in the App Store.
+3. Create CloudPay packages via the Admin console that match the IAP packages in terms of cost, type (renewable or non-renewable) and subscription length.
```

Comment:
maybe CloudPay Admin console and bellow is with lower case. I am not exactly right person to say but better to be the same naming